src/components/App.js

```diff
@@ -13,15 +13,15 @@ import Routes from 'console/components/Routes';
 import CircleLogo from 'console/components/svg/CircleLogo';
 import { notifyAuthenticationError } from 'console/state/auth/actions';
 import { getError } from 'console/state/auth/selectors';
-import { getCurrentRouteTree } from 'console/state/router/selectors';
+import { getCurrentDocumentTitle } from 'console/state/router/selectors';
 import { reverse } from 'console/urls';
 
 const { Header } = Layout;
 
 @connect(
   (state, props) => ({
     authError: getError(state),
-    routeTree: getCurrentRouteTree(state),
+    documentTitle: getCurrentDocumentTitle(state, 'Delivery Console'),
```

Comment:
Do you plan on having anything else than `Delivery Console` here? Could this argument be dropped out, or be added as the default `defaultsTo`?

Reply:
I'm not sure about this. I made it like this arbitrarily, because both ways make sense to me.